src/com/lilithsthrone/game/character/race/RacialBody.java

```diff
@@ -637,30 +637,14 @@ public boolean isHairTypeLinkedToFaceType() {
 			SkinType.HARPY,
 			BodyMaterial.FLESH,
 			HornLength.ZERO_TINY, HornLength.ZERO_TINY,
-			Util.newArrayListOfValues(HornType.NONE), PenisType.AVIAN, 2,
+			Util.newArrayListOfValues(HornType.NONE), PenisType.AVIAN, 5,
 			PenisGirth.ONE_THIN, TesticleSize.ZERO_VESTIGIAL, 2,
 			CumProduction.ONE_TRICKLE, Util.newArrayListOfValues(TailType.HARPY), TentacleType.NONE,
 			VaginaType.HARPY,
 			Wetness.THREE_WET,
 			Capacity.TWO_TIGHT, ClitorisSize.ZERO_AVERAGE, OrificeElasticity.FOUR_LIMBER, OrificePlasticity.THREE_RESILIENT,
 			Util.newArrayListOfValues(WingType.NONE), WingSize.ZERO_TINY, WingSize.ZERO_TINY,
 			GenitalArrangement.CLOACA) {
-
```

Comment:
Thank you for the PR! I'll merge it locally and restore this section, as I don't know why it was removed. ^^